### PR TITLE
Interactive Profiles + bug fixes

### LINKED ISSUE #67: Invalid URLs in seeds file
Hi,
crawl doesn't start if there is even one 'Invalid URL' in seeds file, for example 

yle.fi 
instead of 
https://yle.fi

Crawler should jump over invalid URLs, log them in errors and continue with valid URLs?

### PR DESCRIPTION
**Pull request by ikreymer**
- Initial support for creating profiles interactively via devtools UI
- 
* Misc improvements:
- Bug fixes: ensure screencasting errors are caught and do not interrupt crawl
- Seed validation: print invalid seed urls and exit (fixes #67)
- Optimization: don't extract links if seed depth is already at maximum
- Simplify link extraction logic

* Update to browsertrix-behaviors 0.2.3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -319,13 +319,13 @@ will start a crawl with 3 workers, and show the screen of each of the workers fr
 Browsertrix Crawler also includes a way to use existing browser profiles when running a crawl. This allows pre-configuring the browser, such as by logging in
 to certain sites or setting other settings, and running a crawl exactly with those settings. By creating a logged in profile, the actual login credentials are not included in the crawl, only (temporary) session cookies.
 
-Browsertrix Crawler currently includes a script to login to a single website with supplied credentials and then save the profile.
-It can also take a screenshot so you can check if the login succeeded. The `--url` parameter should specify the URL of a login page.
+Browsertrix Crawler includes a script to login to a single website with supplied credentials and then save the profile, as well as a new 'interactive' profile creation mode.
```
Comment (emmadickson) [inline README.md:322]: much better wording

File: crawler.js [hunk 2]
```diff
@@ -405,14 +409,30 @@ class Crawler {
   }
 
   async extractLinks(page, seedId, depth, selector = "a[href]") {
-    let results = [];
+    const results = [];
```
Comment (emmadickson) [inline crawler.js:412]: should this be a const if we're pushing results to it? would let me more correct?

File: create-login-profile.js [hunk 3]
```diff
@@ -36,6 +44,10 @@ function cliOpts() {
       type: "boolean",
       default: false,
     },
+
+    "interactive": {
+      describe: "Start in interactive mode!",
```
Comment (emmadickson) [inline create-login-profile.js:49]: we could have the default be false for consistency

File: crawler.js (hunk 2 above)
Comment (ikreymer) [reply crawler.js:412]: the way JS works, adding to the array is still ok for const, it just won't allow re-assigning to the results objects.

File: create-login-profile.js (hunk 3 above)
Comment (ikreymer) [reply create-login-profile.js:49]: ok, will add

**Review by emmadickson — APPROVED** [review]
LGTM

**Pull request merged**